grails-app/domain/com/asinha/domain/Payer.groovy

```diff
@@ -30,17 +30,17 @@ class Payer extends BaseEntity {
     Customer customer
 
     static constraints = {
-        address size: 1..255, nullable: false
-        addressNumber size: 1..255, nullable: false
-        city size: 1..255, nullable: false
-        complement size: 1..255, nullable: true
-        cpfCnpj size: 11..14, nullable: false
-        email email: true, size: 5..255, nullable: false
-        phone size: 1..255, nullable: false
-        name size: 1..255, nullable: false
-        postalCode size: 1..255, nullable: false
-        province size: 1..255, nullable: false
-        state size: 1..255, nullable: false
+        address nullable: false
```

Comment:
Por default é nullable false, então não precisa especificar, só se for nullable true